libs/date/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@ledgerhq/live-date",
```

Comment:
this isolated "date" library does not make sense to me. To me it belongs to the idea of a "portfolio". rangeDates + portfolio of live-common should go together because it's basically designed for the portfolio chart UI.